tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_xary_ops.cpp

```diff
@@ -76,6 +76,7 @@ namespace tt::tt_metal::detail {
             expm1 = exp(x) - 1)doc"
         );
         detail::bind_unary_op(m_tensor, "signbit", signbit, R"doc(Applies the signbit function to the elements of the input tensor ``{0}``.)doc");
+        detail::bind_unary_op(m_tensor, "unary_floor", unary_floor, R"doc(Applies floor to the elements of the input tensor ``{0}``.)doc");
```

Comment:
Is there a reason this function is called unary_floor instead of just floor?

Also add a line that it is not supported on GS.

Reply:
@tt-aho , we already have a scalar floor function in `composite_ops.cpp` under `power_fp` op,
```
Tensor _power_fp(const Tensor& input_a, float exponent, const MemoryConfig& output_mem_config) {
    TT_FATAL(exponent >= 0.0f, "works for positive exponents only");
    const uint32_t exponent_floor = static_cast<uint32_t>(floor(exponent));
    if (static_cast<float>(exponent_floor) == exponent) {
        return power(input_a, exponent_floor, output_mem_config);
    }
```
If we name our floor op as floor, it will show conflict error

Reply:
What is the conflict?
You should be able to refer to the one from c++ lib as std::floor, and the ours as tt::tt_metal::floor